moleculemaker/molli-frontend
# feat: integrate molli-frontend + mmli-backend

## Problem
`mmli-job-manager` no longer sends notification email upon success or failure of CLEAN/MOLLI jobs

## Approach
* feat: integrate molli-frontend + mmli-backend
    * replaces the legacy `mmli-job-manager` with the new backend `mmli-backend`

## How to Test
Must be tested with https://github.com/moleculemaker/mmli-backend/pull/61

1. Navigate to https://molli.frontend.staging.mmli1.ncsa.illinois.edu/configuration
2. Download & unzip [MOLLI-input-files.zip](https://github.com/user-attachments/files/16950172/MOLLI-input-files.zip)
3. Upload CORES File: `BOX_cores_test_1.cdxml`
4. Upload SUBS File: `BOX_substituents_test_1.cdxml`
5. Click on "Get the MOLLI result"
6. Wait for job to complete
    * You should see the [results page](https://molli.frontend.staging.mmli1.ncsa.illinois.edu/results/0e29282aaff74bbe8ab1ca6c35c3dede) load and displays visualizations from your current job
